pywinauto/application.py

```diff
@@ -556,19 +550,21 @@ def _ctrl_identifiers(self):
 
         return control_name_map
 
-    def print_control_identifiers(self, depth = 2):
+    def print_control_identifiers(self, depth = None):

```

Comment:
Printing the whole hierarchy by default looks a bit exaggerated for me. User might get confused with a huge number of prints spat out by the function. Maybe we could leave ```depth=2``` but print an info before the all identifiers that there are other options if more levels are required. Something like:
```"Print out identifiers for {0} levels. Use 'depth' argument to get info about deeper sub-levels".format(depth)```